controllers/bentodeployment_controller.go

```diff
@@ -2255,7 +2255,7 @@ monitoring.options.insecure=true`
 		lastPort++
 		monitorExporterProbePort := lastPort
 
-		monitorExporterImage := "quay.io/bentoml/bentoml-monitor-exporter:0.0.2"
+		monitorExporterImage := "quay.io/bentoml/bentoml-monitor-exporter:0.0.3"
```

Comment:
can be an environment var like these?

https://github.com/bentoml/yatai-deployment/blob/main/helm/yatai-deployment/values.yaml#L112

Reply:
https://github.com/bentoml/yatai-deployment/pull/107/commits/c9ce618e990756b5648ca41fe2e105fd0d8903e0

Reply:
I think you should also add new values to the helm chart values file and modify the helm template yaml like this: https://github.com/bentoml/yatai-deployment/blob/main/helm/yatai-deployment/templates/secret-env.yaml#L14